lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -530,10 +530,10 @@ private function getTablePrimaryKeys(Table $table)
     private function getClassNameForTable($tableName)
     {
         if (isset($this->classNamesForTables[$tableName])) {
-            return $this->namespace . $this->classNamesForTables[$tableName];
+            return $this->namespace.$this->classNamesForTables[$tableName];
         }
 
-        return $this->namespace . Inflector::classify(strtolower($tableName));
+        return $this->namespace.Inflector::classify(strtolower($tableName));
```

Comment:
Please revert changes around the concatenation (`.`) operator: we keep spaces around it.